pychunkedgraph/meshing/meshgen.py

```diff
@@ -1036,7 +1036,7 @@ def get_multi_child_nodes(cg, chunk_id, node_id_subset=None, chunk_bbox_string=F
             fragment.value
             for child_fragments_for_node in node_rows
             for fragment in child_fragments_for_node
-        ]
```

Comment:
have you checked that this is correct? @akhileshh @manuel-castro I suspect that this step is actually trying to concatenate a bunch of irregularly sized arrays.

Reply:
I don't see them being concatenated after this line, it looks like the array is being used as a map of children to check whether there is a single child or not (for skip connections). Think Manuel did this to make use of numpy masking.